py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -63,20 +60,22 @@ def trigger_lambda(self) -> None:
         Invoke the ingestion lambda that will take the files in this batch and
         convert them into a parquette file.
         """
+        process_logger = ProcessLogger(
+            "trigger_lambda",
+            ingestion_size=self.total_size,
+            file_count=len(self.filenames),
+        )
+        process_logger.log_start()
```

Comment:
the pattern for most functions is going to be the same

```
create process logger
log start

try:
    do some stuff
    log complete
except:
    log failure
```

most other logging can just removed.